generators/server/templates/src/test/java/package/web/rest/AccountResourceIT_oauth2.java.ejs

```diff
@@ -72,8 +75,7 @@ import org.springframework.transaction.annotation.Transactional;
 <%_ } else { _%>
 @AutoConfigureMockMvc
 <%_ } _%>
-@WithMockUser(value = TEST_USER_LOGIN)
-@IntegrationTest
+SpringBootTest(classes = { <%= mainClass %>.class, AsyncSyncConfiguration.class, TestSecurityConfiguration.class })
```

Comment:
file and class needs to be renamed for this one since its not an integration test anymore